src/shared/components/DateRangeDropdown/DateRangeDropdown.tsx

```diff
@@ -57,6 +60,7 @@ const DateRangeDropdown: FunctionComponent<DateRangeDropdownProps> = ({
 		await closeDropdown();
 	};
 
+	// TODO: Is tagId necessary?
```

Comment:
wat bedoel je met deze TODO?

Reply:
Wrs omdat `tagId` nergens in de function gebruikt wordt

Reply:
ik denk dat die tagId toegevoegd is om uniform te blijven met de andere filter dropdowns, zodat ze allemaal op dezelfde manier hun filter kunnen verwijderen.